andbible/mods.d/HISB.conf

```
@@ -3,8 +3,8 @@ DataPath=./modules/texts/ztext/HISB/
 Lang=he
 Encoding=UTF-8
 Direction=RtoL
-Version=1.3
-Description=Hebrew with Strong and english translitration. Works properly with And Bible 4.0+
+Version=1.4
```

```suggestion
Version=1.3
```

maybe version update not needed because of this ... after all.